N24DCPT018/BUỔI 2/Hellobaby.cpp

```diff
@@ -0,0 +1,7 @@
+#include<iostream>
+#include<string.h>
+using namespace std;
+int main()
+{
+    cout<<"Hello world";
```

Comment:
Chỗ này em nên thêm kí tự xuống dòng ('\n') hoặc std::endl để console in ra và xuống dòng mới luôn.